.github/workflows/assign.yaml

```diff
@@ -2,12 +2,14 @@
 # out of draft (ready for review). 
 #
 # NOTE: pull_request_target behaves the same as pull_request except it grants a
-# read/write token to workflows running on a pull request from a fork. While this 
-# may seem unsafe, we are limiting the permissions of the Github token below.
+# read/write token to workflows running on a pull request from a fork. While
+# this may seem unsafe, we are limiting the permissions of the Github token
+# below.
 name: Assign
 on: 
   pull_request_target:
     types: [assigned, opened, reopened, ready_for_review]
```

Comment:
I don't think this workflow will assign additional reviewers if the file types change (ex: PR opens with no docs changes then author pushes a commit with docs changes).

Reply:
```suggestion
    types: [assigned, opened, reopened, ready_for_review, synchronize]
```